mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/associations_view.sql

```diff
@@ -22,8 +22,16 @@ CREATE OR REPLACE VIEW associations_view
     FROM get_profile_association_snapshot('action_to_action_profiles', 'action_profiles', 'ACTION_PROFILE', 'action_profiles', 'ACTION_PROFILE')
       UNION ALL
     SELECT association_id, master_id, master, master_type, detail_id, detail_type, detail_order, detail,  react_to, job_profile_id
-    FROM get_profile_association_snapshot('action_to_mapping_profiles', 'action_profiles', 'ACTION_PROFILE', 'mapping_profiles', 'MAPPING_PROFILE');
```

Comment:
To change the associations_view view, update this script and copy it to the appropriate scripts.snippet field of the schema.json

Reply:
I have made the changes are suggested.
Request you to review.